### DISCUSSION TITLE
Gestión de conexiones SSH

### DISCUSSION THREAD

**Discussion by Bizkaitarra** [CodelyTV/dotly#151] [Ideas]
Utilizo sshpass para crearme conexiones. Sé que habrá una manera mejor pero con escribiendo css  (de connect ssh) me salen todas las conexiones ssh que he guardado y en dos clicks estoy en el servidor en cuestión:

function cssh_proyecto_desarrollo() {
    export SSHPASS=mipasss
    sshpass -e ssh -p 27 usuario@servidor.com
}

Sería interesante si existe mejor manera que pudiera estar integrado en dotly

**Comment by gtrabanco** [comment]
Te sugiero que uses llaves publica/privada para automatizar el proceso de conexión a servidores por ssh. Y para conectarte de forma facil a los servidores usar configuraciones ssh en local. Tengo scripts que gestionan esto mismo ya que es algo que hago habitualmente, échale un vistazo a estos scripts:
- https://github.com/gtrabanco/dotfiles/tree/master/scripts/ssh

Esperaba meterlos en el core de dotly pero si conseguimos lanzar el marketplace de scripts estará ahí.

Un saludo.

**Comment by Bizkaitarra** [comment]
Le echaré un ojo, muchas gracias!

**Reply by gtrabanco** [reply]
Si necesitas ayuda dilo que te recomiendo algún artículo o te explico lo que haga falta.

Un saludo.
